### PR TITLE
Add precommits lints setup

### LINKED ISSUE #41: Add precommits and lints to check styles
# REQUIREMENTS
Write husky precommit to check styles on rb, js and sass files. (Rubocop, ESLint and Stylelint)

### PR DESCRIPTION
**Pull request by cardotrejos**
# Description
What did you implemented/Why did you implemented this:
- I added husky-precommits and some lints to check the styles of the staged files.
- This help us to send our changes with better style, we waste time reviewing style (lines, spaces, etc)

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "printWidth": 120,
+    "proseWrap": "preserve",
+    "quoteProps": "as-needed",
+    "requirePragma": false,
+    "semi": true,
+    "singleQuote": true,
+    "tabWidth": 2,
+    "trailingComma": "es5",
+    "useTabs": false
+}
```
Comment (YilKanda) [inline .prettierrc]: A blank space is missing.

File: .stylelintrc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      ignore: ["after-comment", "first-nested"],
+    } ],
+    "scss/dollar-variable-empty-line-before": [ "always", {
+      except: [
+        "after-dollar-variable",
+      ],
+      ignore: ["after-comment"],
+    } ]
+  }
+}
```
Comment (YilKanda) [inline .stylelintrc]: A blank space is missing

File: app/javascript/controllers/maps_controller.js [hunk 3]
```diff
@@ -1,87 +1,87 @@
-import { Controller } from "stimulus"
+import { Controller } from 'stimulus';
 
 export default class extends Controller {
-  static targets = ["map", "field", "getCurrentLocation"]
+  static targets = ['map', 'field', 'getCurrentLocation'];
+
   static values = {
     latitude: Number,
     longitude: Number,
```
Comment (YilKanda) [inline app/javascript/controllers/maps_controller.js]: Please, do `rebase` to master.

File: app/javascript/controllers/maps_controller.js [hunk 4]
```diff
@@ -1,87 +1,87 @@
...
+    this.autocomplete();
   }
 
   map() {
-    if (this._map == undefined) {
+    if (this._map === undefined) {
       this._map = new google.maps.Map(this.mapTarget, {
-        center: new google.maps.LatLng(
-          this.latitudValue,
-          this.longitudValue
```
Comment (YilKanda) [inline app/javascript/controllers/maps_controller.js]: Please, do rebase to master.

**Review by YilKanda — CHANGES_REQUESTED** [review]

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by Ilyeo — APPROVED** [review]

**Review by KarinaDell — APPROVED** [review]

**Review by EduJimenez96 — APPROVED** [review]

**Pull request merged**